cli.js

```diff
@@ -6,14 +6,15 @@ var cpy = require('./');
 var cli = meow({
 	help: [
 		'Usage',
-		'  $ cpy <source> <destination> [--no-overwrite] [--parents] [--cwd <dir>]',
+		'  $ cpy <source> [--rename <filename>] <destination> [--no-overwrite] [--parents] [--cwd <dir>]',
```

Comment:
Can you place the flag at the end for clarity?

I also think we should do `[--rename=<filename>]` instead of `[--rename <filename>]` for clarity.